draft-annevk-johannhof-httpbis-cookies.md

```diff
@@ -1193,11 +1216,24 @@ boolean _httpOnlyAllowed_, boolean _allowNonHostOnlyCookieForPublicSuffix_, and
 
 1. Insert _cookie_ into the user agent's cookie store.
 
-1. Remove all expired cookies from the user agent's cookie store.
+1. Return _cookie_.
```

Comment:
I thought we had identified the need for knowing whether this would change an existing cookie? See https://github.com/WICG/cookie-store/issues/246

Reply:
As discussed offline in our meeting on 2025-02-27: We will check if _cookie_'s attributes match _oldCookie_ and if they do abort Store A Cookie and return null.